src/wymeditor/editor/native-edit-registration.js

```diff
@@ -51,7 +51,9 @@ WYMeditor.NativeEditRegistration.prototype._onAnyNativeEdit = function () {
         undoRedo = nativeEditRegistration.wym.undoRedo;
 
     // remove redo points
-    undoRedo.history.changesetsFore = [];
```

Comment:
This private property was gone in object-history v3.0.0.

This would have been caught had we had proper test coverage for undo.